phalcon/mvc/view/engine/volt/compiler.zep

```diff
@@ -2210,8 +2219,23 @@ class Compiler implements InjectionAwareInterface
 				}
 				let this->_autoescape = autoescape;
 			}
+
+            /**
+             * Configures the compiler to register PHP Functions automatically.
+             */
+            if isset options["includePhpFunctions"] {
+                let includePhpFunctions = options["includePhpFunctions"];
```

Comment:
How about make it possible to select which exactly php functions we want to include? Just accept boolean or array maybe?